src/constants.ts

```diff
@@ -56,4 +56,5 @@ export const nativeTestnetChainIdentifiers: string[] = [
   "test-core",
   "govgen",
   "seda-1-testnet",
+  "xion-testnet",
```

Comment:
Could you remove this line? The registry doesn't require it.

Reply:
Done